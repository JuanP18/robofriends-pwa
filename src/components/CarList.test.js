import React from 'react';
import {shallow} from 'enzyme';
import CardList from './CardList';

describe('Card List Component', () => {
  it('expect to render CardList component', () => {
    const mockRobot = [{
        id:1,
        name: 'Jhon Snow',
        userName: 'JohnJohn',
        email: 'jhon@gamil.com'
    }];  
    expect(shallow(<CardList robots={mockRobot} />)).toMatchSnapshot();
  })
});