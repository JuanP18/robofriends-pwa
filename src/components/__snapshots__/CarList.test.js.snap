// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card List Component expect to render CardList component 1`] = `
<div>
  <Card
    email="jhon@gamil.com"
    id={1}
    key="0"
    name="Jhon Snow"
  />
</div>
`;
